gatsby-config.js

```diff
@@ -73,38 +73,86 @@ const addBreadcrumbsToNodes = (nodes, advocacy = false) => {
   return newNodes;
 };
 
+const splitNodeContent = nodes => {
```

Comment:
Curious why this is needed/what it's doing? Looks like it's splitting up content into smaller chunks to get under some limit?